public/components/common/modules/events-enhance-discover-fields.ts

```diff
@@ -106,7 +106,14 @@ export const EventsEnhanceDiscoverCell = {
   }), {
     contentRegex: /(\w+)/g,
     element: 'span'
-  })
+  }),
+  'syscheck.value_name': (content, rowData, element, options) => {
+    if (content) return;
+    const container = document.createElement("td");
```

Comment:
Maybe a `div` or `span` instead of an HTML related to a table. The `td`/`tr` elements are set by the table itself.

Reply:
Just checked and you were right, replaced the element with a `span`